src/pages/FranceConnect.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Form from '../components/Form';
+import Nav from '../components/Nav';
+
+const FranceConnectFormConfiguration = {
+  provider: 'franceconnect',
+  franceConnected: false,
+  scopes: [],
+  cguLink: '/docs/CGU%20FS%20-%20Corps%20juridique%20v2.6.pdf',
+  text: {
+    title: "Demande d'accès à FranceConnect",
+  },
+};
+
+const IntroDescription = () => (
+  <div className="intro">
+    <p>
+      Pour avoir accès à FranceConnect, vous devez obtenir un agrément.
+      L&apos;accès à ce service n&apos;est pour l&apos;instant disponible que si
+      vous êtes:
+    </p>
+    <ul>
+      <li>une administration</li>
+      <li>
+        une entreprise prestataire d&apos;une administration ou ayant une
```

Comment:
We don't have this on the public website. We whould confer to see which version should be displayed